### DISCUSSION TITLE
Beatmaps can't download from the listing (on Android)

### DISCUSSION THREAD

**Discussion by Bendyg7** [ppy/osu#20666] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When downloading the Beatmap, the loading freeze at 0% and then says "the download failed"

### Screenshots or videos

![Screenshot_2022-10-09-08-27-26-88_25a1a32208bbcdc1d450b7aa854bb161](https://user-images.githubusercontent.com/75980730/194741727-538266b9-1a63-4b67-92e6-b35ea6a364c8.jpg)

### Version

2022.1008.2

### Logs

[database.log](https://github.com/ppy/osu/files/9740896/database.log)
[network.log](https://github.com/ppy/osu/files/9740897/network.log)
[performance.log](https://github.com/ppy/osu/files/9740898/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9740899/runtime.log)

**Comment by Bendyg7** [comment]
And here's the system specifications
![Screenshot_2022-10-09-08-51-50-74_fc704e6b13c4fb26bf5e411f75da84f2](https://user-images.githubusercontent.com/75980730/194742489-e3a5bd60-4a6a-4c2a-83da-1fb131de35f3.jpg)
![Screenshot_2022-10-09-08-51-33-37_fc704e6b13c4fb26bf5e411f75da84f2](https://user-images.githubusercontent.com/75980730/194742491-87a137f3-ebd7-42bb-a237-84a2206921b8.jpg)

**Comment by nagi-desuuu** [comment]
Have you tried on a different network, preferably a stronger one?
Did you check that you're using a VPN service?
And most importantly, have you rebooted your router/device?

**Reply by Bendyg7** [reply]
When I rebooted the router, the problem fixed.
Thank you for informing me that!
And sorry if this report disturbed you

**Comment by bdach** [comment, marked as answer]
Cannot reproduce locally, logs show generic timeout error.

```
2022-10-09 06:27:22 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmapsets/221860/download failed with System.Net.Http.HttpRequestException: Connection timed out ---> System.Net.Sockets.SocketException: Connection timed out
2022-10-09 06:27:22 [verbose]: at System.Net.Http.ConnectHelper.ConnectAsync (System.String host, System.Int32 port, System.Threading.CancellationToken cancellationToken) [0x00110] in <60e6df34c36b4735b4e6bb1476a99af5>:0
```

As above, this looks like a local network issue.

**Reply by Bendyg7** [reply]
Thank you sir!
The problem fixed by rebooting the router
